src/Raven.Server/Documents/Handlers/IndexHandler.cs

```diff
@@ -576,6 +576,11 @@ public async Task Reset()
         {
             var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
 
+            if (LoggingSource.AuditLog.IsInfoEnabled)
+            {
+                LogAuditFor(Database.Name, $"Index {name} RESET");
```

Comment:
As stated in https://issues.hibernatingrhinos.com/issue/RavenDB-22362/Unify-audit-log-lines please put the action done on a resource as the first word e.g.

RESET index {name}